README.md

```diff
@@ -1,2 +1,18 @@
-# nerc-netbox-device-library
-Collection of device and module templates used in NERC Netbox
+# NERC Netbox Device Library
+
+## Description
+This repo contains the collection of device and module templates and provides the workflow actions to publish them to NERC Netbox deployment.
+
+## Usage
+Template format is expected to be consistent to that use by Netbox export to yaml function.
+
+Main branch is synced with production repo. PR to main branch will trigger sync to development install of netbox.
+
+## Limitations
+Manufacturer creation is not supported and is being handled by https://github.com/nerc-project/nerc-netbox repo.
+
+CI/CD does not raise a failed job if template import fails inside the import script.  Ouput of the workflow job needs to be examined to confirm success.
```

Comment:
I guess this is because `nb-dt-import.py` doesn't return a fail error code if the API doesn't expect it? Or is this an API limitation? If it's just the script not returning an error code I'd almost rather fork it and add that functionality to it ourselves. The github action reporting a fail/success state is pretty important, especially in PRs because if it is able to report failure we can set it to block merging the PR.

Reply:
Yes nb-dt-import.py just skips over devices that it can not import.

Besides forking the script, I have also considered making github workflow parse the output of the script to generate an action failure.

Currently, I would manually: 
- review output of the action run
-  check if the devices included in PR made it into dev netbox correctly.  
 
Would you consider manual verification acceptable for the first pass, with improvement in the future PR?

Reply:
Yes, manual verification is fine as a first pass